README.md

```diff
@@ -135,53 +139,53 @@ print("\n--- Day-Ahead Wind and Solar Forecast Data ---")
 print(df.head())
 ```
 
+</details>
 
-## Getting Started
 
-Please follow the [installation procedure](#installation--usage) and then run the following:
+## Documentation 
 
-```python
-from __future__ import print_function
-import time
-import elexonpy
-from elexonpy.rest import ApiException
-from pprint import pprint
-
-# create an instance of the API class
-api_instance = elexonpy.AvailabilityApi(elexonpy.ApiClient(configuration))
-format = 'format_example' # str | Response data format. Use json/xml to include metadata. (optional)
-
-try:
-    # Fourteen-day generation forecast
-    api_instance.generation_availability_summary14_d_get(format=format)
-except ApiException as e:
-    print("Exception when calling AvailabilityApi->generation_availability_summary14_d_get: %s\n" % e)
-
-# create an instance of the API class
-api_instance = elexonpy.AvailabilityApi(elexonpy.ApiClient(configuration))
-format = 'format_example' # str | Response data format. Use json/xml to include metadata. (optional)
-
-try:
-    # Three-year generation forecast
-    api_instance.generation_availability_summary3_yw_get(format=format)
-except ApiException as e:
-    print("Exception when calling AvailabilityApi->generation_availability_summary3_yw_get: %s\n" % e)
-```
-
-## Documentation for API Endpoints
+### API Endpoints
 
 Documentation for the API Endpoints can be found [here](./docs/DocApiEndpointsList.md)
 
-## Documentation For Models
+### Models
 Documentation for the Models can be found [here](./docs/DocModelsList.md)
 
-## Documentation For Authorization
+### Authorization
 
  All endpoints do not require authorization.
 
 
-## Author
+##  FAQ
+
+### How do I get an API key?
+
+You dont need one. The Elexon API does not require an API key for access.
+
+### How do I get a year wroth of data?
+
+You currently have to write a loop yourself. We hope to incorporate this into the package in the future.
+
+## Development
+
+To install the package directly from the GitHub repository, use the following command:
+
+```shell
+ pip install git+https://github.com/openclimatefix/Elexonpy.git
+```
+
+### Tests
+
+To run the tests use `pytest`
```

Comment:
I'd be tempted to put this as a shell command that runs the full suite, e.g.

```bash
$ pytest -s . 
```

or whatever the command actually is!